client/cmd/dexc-desktop/pkg/Build-Windows.md

```diff
@@ -0,0 +1,41 @@
+## Windows build
+
+### Setting up the build environment
+
+This build setup is expected to be run on a freshly installed system (e.g. in a VM), which is the best practice for building binaries for release on Windows, therefore there are no prerequisites.  The setup script will install all components of the toolchain, dependencies and SDKs required to build the `dexc-desktop` binary and the MSI (MS installer), there is no need to install anything manually before running this (it can actually cause issues if different versions of the dependencies are already on the system).  The builder was tested on Windows 10, Windows 11 and server equivalents, 2019/2022.
+
+Download and run setup as Administrator (privileges are required in order to set environment variables).  Open a command prompt as Administrator, and run:
+
+ ```batch
+ cd %UserProfile%
+ curl -O https://raw.githubusercontent.com/decred/dcrdex/master/client/cmd/dexc-desktop/pkg/setup-windows.cmd
+ setup-windows.cmd <branch> <repoUrl>
```

Comment:
On a fresh Windows 10 install
```
C:\Users\buck5>setup-windows.cmd
Installing git...
Installing PowerShell...
Cloning https://github.com/decred/dcrdex@master...
Cloning into 'dcrdex'...
remote: Enumerating objects: 30531, done.
remote: Counting objects: 100% (1661/1661), done.
remote: Compressing objects: 100% (1023/1023), done.
remote: Total 30531 (delta 987), reused 1076 (delta 619), pack-reused 28870
Receiving objects: 100% (30531/30531), 53.70 MiB | 25.19 MiB/s, done.
Resolving deltas:   7% (1591/22728)
Resolving deltas: 100% (22728/22728), done.
The argument 'pkg\setup-windows-stage2.ps1' to the -File parameter does not exist. Provide the path to an existing '.ps1' file as an argument to the -File parameter.
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Try the new cross-platform PowerShell https://aka.ms/pscore6
```

Reply:
`setup-windows.cmd` calls `pkg\setup-windows-stage2.ps1` after cloning the repo; for testing this PR the  fork repo/branch needs to be specified:

`setup-windows.cmd windows-build https://github.com/peterzen/dcrdex`

Reply:
The script should probably explain this fact if the file is not found, I'll add this.